devolearn/embryo_segmentor/embryo_segmentor.py

```diff
@@ -178,7 +180,8 @@ def predict_from_video(self, video_path, pred_size = (350,250), save_folder = "p
             for i in tqdm(range(len(images)), desc = "saving predictions: "):
                 save_name = save_folder + "/" + str(i) + ".jpg"
                 tensor = self.mini_transform(images[i]).unsqueeze(0)
-                res = self.model(tensor).detach().cpu().numpy()[0][0]
+                device_tensor = Variable(tensor.to(self.device))
```

Comment:
@rtharungowda  The `Variable()` wrapper will not be required to move a tensor to the GPU.  Let us not use unnecessary imports where things can be done without them. 

it can just be 
```python
device_tensor = tensor.to(self.device)
```

or better yet, we can just  make it 

```python
tensor = self.mini_transform(images[i]).unsqueeze(0).to(self.device)
res = self.model(tensor).detach().cpu().numpy()[0][0]
```